Source/Core/VideoCommon/DriverDetails.h

```diff
@@ -288,6 +288,16 @@ enum Bug
   // Mali Vulkan driver, causing high CPU usage in the __pi___inval_cache_range kernel
   // function. This flag causes readback buffers to select the coherent type.
   BUG_SLOW_CACHED_READBACK_MEMORY,
+
+  // BUG: Apparently ARM Mali GLSL compiler managed to break bitwise AND operations between
+  // two integers vectors, when one of them is non-constant (though the exact cases of when
+  // this occurs are still unclear to me). The resulting vector from the operation will be
```

Comment:
This description should probably avoid the self-reference here, particularly since it's not clear out of context who "me" is.

Reply:
sure